assets/dns/daemonset.yaml

```diff
@@ -82,9 +82,7 @@ spec:
             path: Corefile
       - name: metrics-tls
         # secretName is set at runtime
-      tolerations:
-      - key: "node-role.kubernetes.io/master"
-        operator: Exists
+      # tolerations is set at runtime.
```

Comment:
Would you mind removing the hard-coded `nodeSelector` in `daemonset.yaml` and adding a comment similar to this one in it's place?

Reply:
Done.